recurly/recurly-js
# Add Croatia to the list of EU countries

On 7/1/2013, Croatia was officially added to the EU (http://europa.eu/newsroom/calendar/event/292084/croatia-joins-the-eu).

So we need to add Croatia to the EU set so that the VAT Number field logic displays.

Approvers: @aheckmann @scwchoi 

Tests:
- Open `examples/subscribe.html`
- Select a non-eu country, notice no VAT field
- Select Croatia, notice a VAT field

Example: No VAT
![recurlyjs subscribe example 2014-02-19 16-10-29](https://f.cloud.github.com/assets/175496/2213664/86f771aa-99c3-11e3-9679-62deeb579b44.png)

Example: Croatia, with VAT
![recurlyjs subscribe example 2014-02-19 16-10-54](https://f.cloud.github.com/assets/175496/2213668/93b5c81a-99c3-11e3-9264-8dbbb025b598.png)
